cf-deployment.yml

```diff
@@ -5,30 +5,30 @@ properties_shared_across_every_instance_group:
     syslog_daemon_config:
       enable: false
     metron_agent:
-      deployment: "((metron_agent_deployment_name))"
+      deployment: "((system_domain))"
```

Comment:
That's how it is in the [cf-filler recipe](https://github.com/cloudfoundry/cf-deployment/blob/master/cf-filler/recipe-cf-deployment.yml#L18).  I set that originally because it seemed like a sensible default, and was one less decision that the user had to make.